iOS/testbed/iOSTestbedTests/iOSTestbedTests.m

```diff
@@ -0,0 +1,125 @@
+#import <XCTest/XCTest.h>
+#import <Python/Python.h>
+
+@interface iOSTestbedTests : XCTestCase
+
+@end
+
+@implementation iOSTestbedTests
+
+
+- (void)testPython {
+    // Arguments to pass into the test suite runner.
+    // argv[0] must identify the process; any subsequent arg
+    // will be handled as if it were an argument to `python -m test`
+    const char *argv[] = {
+        "iOSTestbed", // argv[0] is the process that is running.
+        "-uall,-gui,-curses",  // Enable most resources; GUI and curses tests won't work on iOS
```

Comment:
The curses and tkinter tests will be skipped automatically due to their stdlib modules not being built, so there's no need to exclude them here.